medical_medicament/models/product_template.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, _
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    is_medicament = fields.Boolean(
```

Comment:
A string 'Medicament' as made into `is_vaccine` should be great (I think I had forgotten in the first version :smile: )